### DISCUSSION TITLE
Chaining flycheck checkers

### DISCUSSION THREAD

**Discussion by 3ynm** [emacs-lsp/lsp-mode#3708] [Q&A] [answered]
Is [this](https://github.com/flycheck/flycheck/issues/1762#issuecomment-750458442) the way to chain flycheck checkers when using LSP?

I couldn't find information about this on the docs.

**Comment by yyoncho** [comment, marked as answer]
Yes. The fixes and the discussions should happen on flycheck mode side, there is nothing we can do on lsp-mode's side.
